cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup.test.cpp

```diff
@@ -907,7 +915,7 @@ HEAVY_TYPED_TEST(stdlib_biggroup, compute_naf)
     }
 }
 
-/* These tests only work for UltraComposer */
+/* These tests only work for UltraPlonkComposer */
```

Comment:
This comment seemed to be highlighting that the tests dont run for Standard/Turbo but now this suggest it works for Ultra Plonk and not Ultra Honk - Is this the case?

Reply:
Nice catch. Checked that they are enabled for Ultra